pandas/tests/frame/methods/test_values.py

```diff
@@ -226,7 +226,10 @@ def test_values_lcd(self, mixed_float_frame, mixed_int_frame):
 
 
 class TestPrivateValues:
-    def test_private_values_dt64tz(self, request):
```

Comment:
on the off-chance this PR isn't merged, should remove the request arg here

Reply:
using_array_manager isnt used?

Reply:
Yep, and request also not anymore. Updated